articles/fleet-4.15.0.md

```diff
@@ -0,0 +1,77 @@
+# Fleet 4.15.0 adds beta support for self-service, increases transparency, and brings Zendesk to the party.
```

Comment:
```suggestion
# Fleet 4.15.0 adds beta support for self-service, scope transparency, and brings Zendesk to the party.
```

I think we should use "Scope transparency" to describe the transparency feature. 

This is because Fleet has been using "Scope transparency" to describe, to users, customers and stakeholders, the ability for end users to see what information their organization collects.

Reply:
Should any of these be capitalized because they are the name of the features?

Reply:
@Desmi-Dizney yes, great catch. I think "Self-service" and "Scope transparency" should both be capitalized.